lib/pbench/cli/server/user_management.py

```diff
@@ -0,0 +1,205 @@
+import click
+
+from pbench import BadConfig
+from pbench.cli.server import config_setup, pass_cli_context
+from pbench.cli.server.options import common_options
+from pbench.server.database.database import Database
+from pbench.server.database.models.users import User
+
+
+# User create CLI
+@click.group("user_group")
+@click.version_option()
+@pass_cli_context
+def cli(ctx):
+    pass  # Entry Point
+
+
+@cli.command()
+@common_options
+@click.option(
+    "--username",
+    prompt=True,
+    required=True,
+    help="pbench server account username (will prompt if unspecified)",
+)
+@click.option(
+    "--password",
+    prompt=True,
+    hide_input=True,
+    required=True,
+    help="pbench server account password (will prompt if unspecified)",
+)
+@click.option(
+    "--email",
+    prompt=True,
+    required=True,
+    help="pbench server account email (will prompt if unspecified)",
+)
+@click.option(
+    "--first-name",
+    prompt=True,
+    required=True,
+    help="pbench server account first name (will prompt if unspecified)",
+)
+@click.option(
+    "--last-name",
+    prompt=True,
+    required=True,
+    help="pbench server account last name (will prompt if unspecified)",
+)
+@pass_cli_context
+def user_create(
+    context: object,
+    username: str,
+    password: str,
+    email: str,
+    first_name: str,
+    last_name: str,
+):
```

Comment:
You left off the type hint for the return value -- which is fine with me in this case, except for the fact that you included it for the other CLI functions.  :-D